samples/kubernetes/README.md

```diff
@@ -0,0 +1,17 @@
+# Running in Kubernetes
```

Comment:
This is running in AKS not agnostic Kubernetes - sample should be explicit about that or include just the K8 commands

Reply:
Thanks, good point. Not gone through the readme files yet. Will do that when the config is sorted.